package.json

```diff
@@ -33,8 +33,8 @@
     "docs:watch": "npm run docs:prepare && gitbook serve",
     "docs:publish": "npm run docs:clean && npm run docs:build && ./scripts/publish-docs",
     "flow": "eslint packages",
-    "todo": "leasot 'packages/*/src/**.js'",
-    "todo-staged": "leasot -x --reporter markdown 'packages/*/src/**.js' > TODO.md && git add TODO.md",
+    "build:todos": "leasot -x 'packages/*/src/**.js'",
+    "publish:todos": "leasot -x --reporter markdown 'packages/*/src/**.js' > TODOS.md",
```

Comment:
Publish? We don't publish anything? We just generate them. 😕

Reply:
hah, friggin naming... `build:todos-list`? `build:todos-file`? `docs:todos`?

Reply:
generate:todos just dumps them to stdout
build:todos adds a file

It isn't docs really.